restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -63,19 +62,9 @@
   @Inject private Db dbFactory;
   private static final Logger logger = LoggerFactory.getLogger(DocumentResourceV2.class);
   private static final ObjectMapper mapper = new ObjectMapper();
-  private final DocumentService documentService;
+  @Inject private DocumentService documentService;
   private final int DEFAULT_PAGE_SIZE = DocumentDB.SEARCH_PAGE_SIZE;
```

Comment:
Since we are already here with the dependency injection, can I propose the extraction of configuration options to:

```
interface DocsApiConfiguration {

    default getDefaultPageSize() {
        return DocumentDB.SEARCH_PAGE_SIZE;
    }

}
```

Then you can:
* register the configuration in the DI
* if and when needed fine tune config options in the test by mocking the configuration interface

Makes sense?